pystan/model.py

```diff
@@ -262,8 +262,8 @@ def __init__(self, file=None, charset='utf-8', model_name="anon_model",
             os.path.join(pystan_dir, "stan", "src"),
             os.path.join(pystan_dir, "stan", "lib", "stan_math"),
             os.path.join(pystan_dir, "stan", "lib", "stan_math", "lib", "eigen_3.3.3"),
-            os.path.join(pystan_dir, "stan", "lib", "stan_math", "lib", "boost_1.66.0"),
-            os.path.join(pystan_dir, "stan", "lib", "stan_math", "lib", "sundials_3.1.0", "include"),
+            os.path.join(pystan_dir, "stan", "lib", "stan_math", "lib", "boost_1.69.0"),
+            os.path.join(pystan_dir, "stan", "lib", "stan_math", "lib", "sundials_4.1.0", "include"),
```

Comment:
Should we add `opencl` folder here too

Reply:
Perhaps we should wait to see what RStan does. Cmdstan 2.19 does not add opencl by default.

Reply:
Sounds good. We really need to test it before release.